apps/api/src/routers/admin.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime
+from logging import getLogger
+from typing import Any, Optional
+
+from fastapi import APIRouter, Body, Depends, HTTPException, status
+from pydantic import BaseModel, Field, TypeAdapter, ValidationError
+
+from auth.authorization import require_role
+from auth.user_identity import User, utc_now
+from models.ApplicationData import Decision, Review
+from services import mongodb_handler
+from services.mongodb_handler import BaseRecord, Collection
+from utils.user_record import Applicant, Role
+
+log = getLogger(__name__)
+
+router = APIRouter()
+
+ADMIN_ROLES = (Role.DIRECTOR, Role.REVIEWER)
+
+
+class ApplicationDataSummary(BaseModel):
+    first_name: str
+    last_name: str
+    university: str
```

Comment:
Issue: we changed the name of this field in `ApplicationData` from `university` to `school`, so we'll need to use the new name.